include/quicly/cc.h

```diff
@@ -33,20 +33,52 @@ extern "C" {
 #include <assert.h>
 #include <stdint.h>
 #include <string.h>
+#include <time.h>
 #include "quicly/constants.h"
+#include "quicly/loss.h"
 
 typedef enum {
     /**
      * Reno, with 0.7 beta reduction
      */
-    CC_RENO_MODIFIED
+    CC_RENO_MODIFIED,
+    /**
+     * CUBIC (RFC 8312)
+     */
+    CC_CUBIC
 } quicly_cc_type_t;
 
+typedef struct st_quicly_cc_conf_t {
+    /**
+     * Congestion controller type.
+     */
+    quicly_cc_type_t type;
+} quicly_cc_conf_t;
+
+#define QUICLY_CC_SPEC_CONF                                                                                                        \
+    {                                                                                                                              \
+        CC_RENO_MODIFIED /* type */                                                                                                \
+    }
+
+#define QUICLY_CC_PERFORMANT_CONF                                                                                                  \
+    {                                                                                                                              \
+        CC_RENO_MODIFIED /* type */                                                                                                \
```

Comment:
Chose Reno for the performance conf for now for backwards compatibility and since the CUBIC implementation is new. As far as I know though most modern OS use CUBIC for TCP by default, so it might be a good idea to switch over down the line.